daemon/images/image_exporter.go

```diff
@@ -44,7 +51,10 @@ func (i *ImageService) PerformWithBaseFS(ctx context.Context, c *container.Conta
 // LoadImage uploads a set of images into the repository. This is the
 // complement of ExportImage.  The input stream is an uncompressed tar
 // ball containing images and metadata.
-func (i *ImageService) LoadImage(ctx context.Context, inTar io.ReadCloser, outStream io.Writer, quiet bool) error {
+func (i *ImageService) LoadImage(ctx context.Context, inTar io.ReadCloser, platform *ocispec.Platform, outStream io.Writer, quiet bool) error {
+	if platform != nil {
+		return errdefs.NotImplemented(errors.New("platform parameter is not supported with the graphdriver image store"))
+	}
```

Comment:
Same here


☝️ ❓ wondering now - could we somehow actually implement this? i.e. _if_ a platform is provided and it matches then proceed - if it doesn't matches, then produce the same error as we do if we have multi-arch support (containerd image store) enabled?